src/sdk/accountspace_fs.js

```diff
@@ -419,10 +402,7 @@ class AccountSpaceFS {
             const account_string = JSON.stringify(requested_account_encrypted);
             nsfs_schema_utils.validate_account_schema(JSON.parse(account_string));
             const name_for_access_key = params.username ?? requester.name;
-            const account_config_path = this._get_account_config_path(name_for_access_key);
-            await native_fs_utils.update_config_file(this.fs_context, this.accounts_dir,
-                account_config_path, account_string);
-            await nb_native().fs.unlink(this.fs_context, requested_account_path);
+            await this.configFS.update_account_config_file(name_for_access_key, JSON.parse(account_string), false, [params.access_key]);
```

Comment:
I don't understand the "false" should it be undefined? the function statement is:
`update_account_config_file(account_name, account_new_data, new_access_keys_to_access_key = [], old_access_keys_to_remove = [])`